src/docs/ws.json

```diff
@@ -0,0 +1,122 @@
+{
+  "asyncapi": "1.0.0",
+  "info": {
+    "title": "Angular NgRx Websocket Endpoint",
+    "version": "1.0.0",
+    "description": "Documentation for the Angular NgRx Websocket Endpoint"
+  },
+
+  "servers": [
+    {
+      "url": "http://localhost:{port}/chat/{chatId}/send",
+      "description": "Allows you to connect using Websocket.",
+      "schema": "ws",
+      "variables": {
+        "chatId": {
+          "default": "397fed73-4bcb-49ea-938b-74d4fe5ce811",
+          "description": "You can get your `chatId` from the get chats REST endpoint"
+        },
+        "port": {
+          "default": "8080"
+        }
+      }
+    },
+
+    {
+      "url": "https://dev.angular-ngrx-chat.com/chat/{chatId}/send",
```

Comment:
Is this a repeat of the previous entry?

Reply:
sort of but the first is the local server and this dev